examples/purchaseTesterTypescript/android/build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
         buildToolsVersion = "33.0.0"
         minSdkVersion = 24
         compileSdkVersion = 34
-        targetSdkVersion = 34
+        targetSdkVersion = 33
```

Comment:
I'm not sure if this is needed but I was getting 
`One of RECEIVER_EXPORTED or RECEIVER_NOT_EXPORTED should be specified when a receiver isn't being registered exclusively for system broadcasts` otherwise... Need to revisit this.

Reply:
I believe this happens when pointing to an android 34 emulator... Seems like it's a problem with a react native dependency not supporting android 34 I would say... Need to look into it, but we can downgrade this for now to get this merged.